### PR TITLE
Delete method of bookbag

### PR DESCRIPTION
**Pull request by MaazMS**
add delete method of bookbag

### CODE REVIEW

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-28-020dac85`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-28-224003b2`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-28-663eaeb1`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-28-21231dec`

**Comment by stakater-user** [comment]
@MaazMS Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/workshop-operator/commit/ab76ec9fa3a555487803c499ea0b57e22db944d7/checks) has Failed!

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-28-acac29c8`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-28-0239f0e6`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-28-589bfdd3`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-28-16a7d022`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-28-0c90bb57`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-28-99ac9b0c`

File: controllers/bookbag_reconciler.go [hunk 1]
```diff
@@ -18,18 +18,28 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+const (
+	BOOKBAG_NAMESPACE_NAME    = "workshop-guides"
+	BOOKBAG_ROLE_BINDING_NAME = "adim"
+	BOOKBAG_ROLE_KIND_NAME    = "Role"
+	BOOKBAG_ROUTE             = 10080
```
Comment (rasheedamir) [inline controllers/bookbag_reconciler.go]: not a good name

**Review by rasheedamir — CHANGES_REQUESTED** [review]
check comments

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-28-e6e2a77d`

**Review by rasheedamir — APPROVED** [review]

**Review by rasheedamir — APPROVED** [review]

**Pull request merged**